resources/js/src/app/components/newsletter/NewsletterInput.vue

```diff
@@ -80,7 +81,9 @@ export default {
             email: "",
             isDisabled: false,
             privacyPolicyValue: false,
-            honeypot: ""
+            honeypot: "",
+            googleRecaptchaApiKey: App.config.global.googleRecaptchaApiKey,
+            loadRecapcha: true
```

Comment:
ist das jemals `false`?

Reply:
Falls es noch benötigt wird, hast du hier noch eine Typo: es fehlt das "t" in "loadRecap**t**cha".